ui/app/views/fusor_ui/layouts/placeholder_base.html.erb

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html lang="<%= I18n.locale %>" class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html lang="<%= I18n.locale %>" class="no-js lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html lang="<%= I18n.locale %>" class="no-js lt-ie9"> <![endif]-->
+<!--[if gt IE 8]><!--> <html lang="<%= I18n.locale %>" class="no-js"> <!--<![endif]-->
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <title><%= h(yield(:title) || "Foreman") %></title>
+    <meta name="description" content="">
+    <meta name="viewport" content="user-scalable=no, width=device-width, initial-scale=1.0, maximum-scale=1.0"/>
+
+    <%= favicon_link_tag "favicon.ico"%>
+
+    <%= stylesheet_link_tag 'application', "data-turbolinks-track" => true %>
+    <%= yield(:stylesheets) %>
+
+    <%= csrf_meta_tags %>
+    <%= javascript_include_tag 'application_no_tl', "data-turbolinks-track" => true %>
+    <%= yield(:javascripts) %>
+
+    <script type="text/javascript">
+      var URL_PREFIX = '<%= request.script_name %>';
+
+      <% if protect_against_forgery? %>
+        var AUTH_TOKEN = '<%= form_authenticity_token.inspect %>';
+      <% else %>
+        var AUTH_TOKEN = null;
+      <% end %>
+
+      <% if SETTINGS[:mark_translated] && !Rails.env.test? %>
+        var I18N_MARK = true;
+      <% end %>
+
+      <%= yield(:inline_javascripts) %>
+    </script>
+
+    <%= yield(:head) %>
+    <!-- begin - Explicitly including themes.  Usually the theme will overload the base layout, but now that we're not using those, we need to add them -->
```

Comment:
Had to include themes manually since this usually overrides the application base layout in foreman which we no longer use.